src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Lottery(String color, int number) {
+        super(color, number);
+    }
+
+    public Ball getRundomBall() {
+        int number = new Random().nextInt(100);
+
+        return new Ball("" + getRandomColor(), number);
+    }
```

Comment:
it should call a `colorSupplier.getRandomColor()` and a random number, create and return a new Ball instance